back/app/controllers/web_api/v1/ideas_controller.rb

```diff
@@ -142,6 +142,9 @@ def create
       input.creation_phase = (participation_context if participation_method.form_in_phase?)
       input.phase_ids = [participation_context.id] if phase_ids.empty?
     end
+    if participation_context.native_survey? && participation_context.allow_anonymous_participation?
```

Comment:
Why do we need the `native_survey?` here? Can we remove it?

Reply:
I think it's a bit messy to give `participation_context.allow_anonymous_participation?` two different meanings. But if we really want to go ahead with this, I think it makes it important to mention this explicitly in the participation methods. The participation methods serve as an overview of the differences, which is helpful in the context of the unified data.